meta-lmp-base/recipes-bsp/u-boot/u-boot-ostree-scr-fit/boot-common.cmd.in

```diff
@@ -97,6 +100,18 @@ if test "${fiovb.debug}" = "1"; then
 	echo "${fio_msg} ###########################################"
 fi
 
+if test -n "${fiovb_rpmb}"; then
+	if test "${fiovb.rollback_protection}" = "1"; then
+		echo "${fio_msg} Anti-rollback protection for boot firmware is enabled"
+		if test ${fiovb.bootfirmware_version} -gt ${dt_bootfirmware_version}; then
+			echo "${fio_msg} Error: It's impossible to downgrade to an older firmware, boot is aborted"
+			echo "${fio_msg} Error: Currently booted firmware: ${dt_bootfirmware_version}, previously booted: ${fiovb.bootfirmware_version} "
+			sleep 5
+			reset
```

Comment:
Sorry, I don't get it. What will be changed after reboot to prevent running this code again?

Reply:
When secondary path is loaded, it will fallback to primary.
If primary is loaded (which basically means that user has manually flashed boot images to primary path) - it will prevent boot and it will lead to infinite reboots.

Reply:
Technically this code should not be called under normal circumstances. If it's called, it means that ak-lite sanity check (boot firmware from new sysroot should have higher version than version of firmware currently installed) was bypassed

Reply:
Ah, so infinite reboots are correct behavior. Ok, thanks.

Reply:
@ricardosalveti if user has flashed manully downgraded boot firmware to primary path when rollback protection is activated, what should we do?